test/test_sbol2_sbol3_direct.py

```diff
@@ -244,6 +253,82 @@ def test_seq_componentDefinition(self):
                 tmp2 = Path(tmpdir) / 'doc2_loop.xml'
                 doc2_loop.write(tmp2)
                 self.assertFalse(file_diff(str(tmp2), str(TEST_FILES / 'seq_componentDefinition.xml')))
+
+    def handle_2to3_conversion(self, test_filename, rubric_filename):
+        """Provides a re-usable test handler for converting SBOL2 to SBOL3 with different test files
+        test_filename: Name of an SBOL2 test file
```

Comment:
Suggest changing the parameter declarations in the docstring to standard format (both here and elsewhere), so they'll link well in IDEs:
```suggestion
        :param test_filename: Name of an SBOL2 test file
```